add Komikku fork (#151)
mihonapp/website: website/src/forks/Komikku/index.md

---
title: Komikku
titleTemplate: false
description: A fork with features from Sy and Mihon plus added personal flair.

layout: home
pageClass: page-komikku

hero:
  name: Komikku

>   text: Features packed

Reviewer: ```suggestion
  text: Feature-packed
```

  tagline: A fork with features from Sy and Mihon plus added personal flair.
  image: /forks/logo-komikku.webp
  actions:
    - theme: brand
      text: Download
      link: https://github.com/komikku-app/komikku/releases/latest
    - theme: alt
      text: GitHub
      link: https://github.com/komikku-app/komikku
    - theme: alt
      text: Preview
      link: https://github.com/komikku-app/komikku-preview/releases/latest

customMetaTitle: Komikku

features:
  - title: Website Suggestions

>     details: Show suggestions / recommendations for current entry from its own website, support all sources.

Reviewer: ```suggestion
    details: Show suggestions and recommendations for the current entry from its own website, supports all sources.
```

    icon: 📚
  - title: Fast Browsing

>     details: Super fast Browsing / Global Search (for whom with large database experiencing slow loading).

Reviewer: ```suggestion
    details: Super fast Browsing and Global Search (for whom with large database experiencing slow loading).
```

    icon: 🏎️
  - title: Hidden Categories

>     details: Hide yours <b>things</b> from <i>nosy</i> people.

Reviewer: ```suggestion
    details: Hides your <b>things</b> from <i>nosy</i> people.
```

    icon: 🕵🏻‍♂️
  - title: Custom Theme

>     details: Unleash your creativity with a fully customizable theme (also come with colorful presets) for endless color lovers.

Reviewer: ```suggestion
    details: Unleash your creativity with a fully customizable theme (also comes with colorful presets) for endless color lovers.
```
